### PR TITLE
Add pylint-plugin-utils

### PR DESCRIPTION
**Pull request by jakirkham**
Generated with `conda skeleton pypi` and cleaned up after.

### CODE REVIEW

File: recipes/pylint-plugin-utils/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    - pylint
+    - astroid
+
+test:
+  imports:
+    - pylint_plugin_utils
+
+about:
+  home: https://github.com/landscapeio/pylint-plugin-utils
+  license: GPL 2
```
Comment (jakirkham) [inline recipes/pylint-plugin-utils/meta.yaml]: Proposing license file be included in PR ( https://github.com/landscapeio/pylint-plugin-utils/pull/4 ).

File: recipes/pylint-plugin-utils/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
+{% set name = "pylint-plugin-utils" %}
+{% set version = "0.2.4" %}
+{% set sha256 = "6844b4d0525aa41ff69c5d44c6edc640e3dab98446abeb499501757f09a3cd71" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  {% set pypi_name = name.replace("_", "-") %}
```
Comment (msarahan) [inline recipes/pylint-plugin-utils/meta.yaml]: I propose that we keep variable setting up at the top, just for easier locatability.  I see why you moved it down in this particular case.  Not a big deal if you disagree.  I'm not terribly strong-feeling on it.

File: recipes/pylint-plugin-utils/meta.yaml (hunk 1 above)
Comment (msarahan) [inline recipes/pylint-plugin-utils/meta.yaml]: Do you want to wait on a merge until then?

Comment (jakirkham) [inline recipes/pylint-plugin-utils/meta.yaml]: Nah, I can add handle this in the feedstock.

File: recipes/pylint-plugin-utils/meta.yaml (hunk 2 above)
Comment (jakirkham) [inline recipes/pylint-plugin-utils/meta.yaml]: Oops, that was suppose to be removed. Thanks for catching that.

**Comment by msarahan** [comment]
thanks!

**Comment by jakirkham** [comment]
@carlio, just letting you know that we have added `pylint-plugin-utils` to [conda-forge]( http://conda-forge.github.io/ ). This will allow users to easily install this with [`conda`]( http://conda.pydata.org/docs/ ) ( [`conda install -c conda-forge pylint-plugin-utils`]( http://anaconda.org/conda-forge/pylint-plugin-utils ) ). The maintenance burden here is light as all releases are built and deployed automatically with CI. Changes to how it is being built can be controlled by changing the recipe in [this feedstock (repo)]( https://github.com/conda-forge/pylint-plugin-utils-feedstock ). If any of you have any interest in being added as a maintainer (to make new releases for instance), we would be more than happy to add any of you to the maintainers list. Thanks for your work on this great project. 😄

**Pull request merged**